### DISCUSSION TITLE
FreeRTOS for Arduino Boards | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#342] [Q&A]
# FreeRTOS for Arduino Boards | Seeed Studio Wiki

FreeRTOS for Arduino Boards

https://wiki.seeedstudio.com/Software-FreeRTOS/

**Comment by BrewingGeek** [comment]
NO graphics with the graphic example..  Also dependent on with version of arduino is used one will NOT compile

**Reply by MatthewJeffson** [reply]
Hi! Thank you very much for your reminder. In fact, many of our wikis have not been updated for a long time. I will write this down and record it in our plans. 
If you have good ideas, you are welcome to submit([pull request](https://wiki.seeedstudio.com/Contribution-Guide/)) them to us.
